cba_settings_userconfig/cba_settings.sqf

```diff
@@ -305,9 +305,9 @@ ace_interact_menu_moveToRoot__ACE_TeamManagement__diwako_dui_buddy_buddy_action_
 
 // ACE Logistics
 force force ace_cargo_enable = true;
-ace_cargo_enableRename = true;
+force force ace_cargo_enableRename = false;
 force force ace_cargo_loadTimeCoefficient = 5;
-ace_cargo_openAfterUnload = 0;
+force force ace_cargo_openAfterUnload = 0;
```

Comment:
Again, this seems like an unnecessary regression to what was previously player-settable.